Library/UUIDType.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 public protocol UUIDType {
   var uuidString: String { get }
+  init()
```

Comment:
so this protocol represents the _type_ and not the instance of the type, so that allows to instantiate a new `UUID()` or `MockUUID()` in our view models which helps to make assertions based on the ID. If we called `UUID()` directly we'd of course get a random `UUID` each time.